vmicore/src/lib/os/linux/ActiveProcessesSupervisor.cpp

```diff
@@ -208,4 +230,25 @@ namespace VmiCore::Linux
         }
         return std::make_unique<std::string>(substringStartIterator, path.cend());
     }
+
+    std::tuple<int, int, int> ActiveProcessesSupervisor::extractKernelVersion() const
```

Comment:
I think the error case of this function should be flipped: Always try to use PTI unless you can exclude it via the banner

Reply:
Let's simply fail if we are unable to extract the banner. No more guessing games.